source/backend/discovery/test/fixtures/additionalResources/dynamodb/stream.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$constants": {
+    "accountId": "xxxxxxxxxxxx",
+    "region": "eu-west-2"
+  },
+  "stream": {
+    "id": "${stream.arn}",
+    "accountId": "${$constants.accountId}",
+    "awsRegion": "${$constants.region}",
+    "availabilityZone": "Not Applicable",
+    "arn": "arn:aws:dynamodb:${$constants.region}:${$constants.accountId}:table/test/stream",
+    "resourceId": "${stream.arn}",
+    "resourceName":  "${stream.arn}",
+    "resourceType": "AWS::DynamoDB::Stream",
+    "relationships": [],
+    "configuration": {}
```

Comment:
we should add a field here to make sure the configuration object is being passed through by doing an assertion for the field in the test